src/backend/stream.rs

```diff
@@ -395,6 +396,7 @@ impl<'ctx> PulseStream<'ctx> {
                 let p = input_buffer_manager.get_linear_input_data(nsamples_input);
                 stm.trigger_user_callback(p, nbytes);
             } else {
+                stm.output_frame_count.fetch_add(nframes, Ordering::SeqCst);
```

Comment:
nit: I'll prefer moving `stm.output_frame_count.fetch_add(nframes, Ordering::SeqCst)` to the next line of `let nframes = nbytes / stm.output_sample_spec.frame_size()` so it become more obvious that it's called as long as we have an output device.